src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
```

Comment:
There should be a space between the class name and the opening brace to follow Java coding conventions.

Reply:
Fixed